### ISSUE TITLE
Getting Started with Triggers :  Missing service/el-hello-listener definition

### ISSUE THREAD

**Issue by bluedog13** [tektoncd/website#471]
In the section: [Running the Trigger](https://tekton.dev/docs/getting-started/triggers/#running-the-trigger), for the below line of code
```
kubectl port-forward service/el-hello-listener 8080
```

**The service/el-hello-listener is not defined/declared in the documentation**.  I don't see service/el-hello-listener created by default in the k8s cluster either by running all the previous steps mentioned in the documentation either. 

Is the service declaration missing from the documentation page? 

Thanks

**Comment by AlanGreene** [comment]
@bluedog13 Are you still having this issue?

The service is created automatically for the EventListener and should be present shortly after you run the `kubectl apply -f event-listener.yaml` command.

What version of Triggers are you running?
What Kubernetes version?
Are the triggers pods in the tekton-pipelines namespace running correctly?
Any errors in their logs?
Did you get any errors on any of the previous steps?
Check `kubectl get events`.

I just ran through the steps from the beginning and the service was created as expected once I applied the event-listener.yaml, I was the able to port-forward as described and received the expected response from the curl request.
